src/UI/FrontEnd.java

```diff
@@ -201,51 +240,75 @@ private void initializeWebAccountModifySection(JPanel websiteAccounts) {
 		JLabel websiteAccountModifyLabel = new JLabel("Modify Account Information");
 		websiteAccountModifyLabel.setBounds(493, 220, 209, 16);
 		websiteAccounts.add(websiteAccountModifyLabel);
-		
+
 		JLabel websiteAccountModifyCurrNicknameLabel = new JLabel("Current Nickname*:");
 		websiteAccountModifyCurrNicknameLabel.setFont(new Font("Lucida Grande", Font.PLAIN, 12));
 		websiteAccountModifyCurrNicknameLabel.setBounds(423, 256, 120, 16);
 		websiteAccounts.add(websiteAccountModifyCurrNicknameLabel);
-		
+
 		websiteAccountModifyCurrNicknameInput = new JTextField();
 		websiteAccountModifyCurrNicknameInput.setColumns(10);
 		websiteAccountModifyCurrNicknameInput.setBounds(541, 251, 130, 26);
 		websiteAccounts.add(websiteAccountModifyCurrNicknameInput);
-		
+
 		JLabel websiteAccountModifyNicknameLabel = new JLabel("New Nickname:");
 		websiteAccountModifyNicknameLabel.setFont(new Font("Lucida Grande", Font.PLAIN, 12));
 		websiteAccountModifyNicknameLabel.setBounds(445, 288, 98, 16);
 		websiteAccounts.add(websiteAccountModifyNicknameLabel);
-		
-		wesbiteAccountModifyNicknameInput = new JTextField();
-		wesbiteAccountModifyNicknameInput.setColumns(10);
-		wesbiteAccountModifyNicknameInput.setBounds(541, 283, 130, 26);
-		websiteAccounts.add(wesbiteAccountModifyNicknameInput);
-		
+
+		websiteAccountModifyNicknameInput = new JTextField();
+		websiteAccountModifyNicknameInput.setColumns(10);
+		websiteAccountModifyNicknameInput.setBounds(541, 283, 130, 26);
+		websiteAccounts.add(websiteAccountModifyNicknameInput);
+
 		JLabel websiteAccountModifyUsernameLabel = new JLabel("New Username:");
 		websiteAccountModifyUsernameLabel.setFont(new Font("Lucida Grande", Font.PLAIN, 12));
 		websiteAccountModifyUsernameLabel.setBounds(445, 321, 98, 16);
 		websiteAccounts.add(websiteAccountModifyUsernameLabel);
-		
+
 		websiteAccountModifyUsernameInput = new JTextField();
 		websiteAccountModifyUsernameInput.setColumns(10);
 		websiteAccountModifyUsernameInput.setBounds(541, 316, 130, 26);
 		websiteAccounts.add(websiteAccountModifyUsernameInput);
-		
+
 		JButton websiteAccountModifyButton = new JButton("Modify");
+		websiteAccountModifyButton.addActionListener(new ActionListener() {
+			public void actionPerformed(ActionEvent e) {
+				if(websiteAccountModifyCurrNicknameInput.getText().isEmpty()) {
+					JOptionPane.showMessageDialog(frame, "Please enter the nickname of the account you want to modify");
+				}else {
+
+					if(UIController.modifyWebsiteAccount(websiteAccountModifyCurrNicknameInput.getText(),websiteAccountModifyNicknameInput.getText(),websiteAccountModifyUsernameInput.getText(),String.valueOf(websiteAccountModifyPasswordInput.getPassword()))) {
+						JOptionPane.showMessageDialog(frame, "Website modified ");
+						
+						String siteNickname = "f";
+						if(websiteAccountModifyNicknameInput.getText().isEmpty()) {
+						siteNickname = websiteAccountModifyCurrNicknameInput.getText(); 
```

Comment:
should this be an error message to the user instead of setting it to its current name and updating website fields even though there isn't anything here to update

Reply:
The user doesn't have to update the nickname, so if they just updated the password or login info then the username would stay the same, the update would be successful, then it would display the changes still

Reply:
Ah, perfect